ballerina/tests/11-custom-object-type-test.bal

```diff
@@ -371,13 +370,105 @@ isolated function selectObjectTypeWithInvalidTypedRecord() returns error? {
     stream<InvalidObjectRecordType, error?> streamData = oracledbClient->query(
         "SELECT pk, col_object FROM TestObjectTypeTable WHERE pk = 1");
     record {}|error? returnData = streamData.next();
+    check streamData.close();
+    check oracledbClient.close();
     if (returnData is sql:ApplicationError) {
         test:assertTrue(returnData.message().includes("Error while retrieving data for unsupported type"),
             "Incorrect error message");
     } else {
         test:assertFail("Querying custom type with invalid record field type should fail with " +
                             "sql:ApplicationError");
     }
+}
+
+@test:Config {
+    groups: ["nested-table"]
+}
+isolated function insertToNestedTable() returns error? {
+        string[] nameAtt = ["Smith", "John", "Arya", "Stark"];
+        NestedTableValue students = new({name: "NestedNameTable", elements: nameAtt});
+        int[] gradesAtt = [67, 45, 78, 86];
+        NestedTableValue grades = new({name: "NestedGradeTable", elements: gradesAtt});
+        int total = 4;
+        int pk = 2;
+        string teacher = "Kate Anderson";
+        sql:ParameterizedQuery insertQuery = `INSERT INTO NestedClassTable(pk, col_teacher, col_students, col_grades, col_total)
+                VALUES (${pk}, ${teacher}, ${students}, ${grades}, ${total})`;
+        sql:ExecutionResult result = check executeQuery(insertQuery);
+        test:assertExactEquals(result.affectedRowCount, 1, "Affected row count is different.");
+        var insertId = result.lastInsertId;
```

Comment:
Format

Reply:
done in https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/32/commits/4eeb31cc93af8617db71bac6ca0db26dc62e0863